src/vs/workbench/services/files/node/fileService.ts

```diff
@@ -120,10 +127,14 @@ export class FileService implements IFileService {
 		}
 
 		if (this.basePath && !this.options.disableWatcher) {
-			if (isWindows) {
-				this.setupWin32WorkspaceWatching();
+			if (this.options.useNsfwFileWatcher) {
```

Comment:
@Tyriar yes this is something I have to follow up with Sandeep, currently multi-folder comes in late.